api/app/controllers/review.controller.ts

```diff
@@ -251,6 +251,29 @@ export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
     }[];
   } = {};
 
+  if (status) {
+    const currentTime = new Date().getTime();
+    switch (status) {
+      case QUERY_REVIEW_STATUS.ACTIVE:
+        const activeCondition = buildIsActiveCondition(currentTime);
+        condition[Op.and as unknown as keyof typeof Op] = activeCondition;
+        break;
+
+      case "expired":
```

Comment:
I think the "expired" can be scoped to constant.

Reply:
I don't think that's necessary as we only use this here. Its not used anywhere else in the code.